seed/challenges/01-front-end-development-certification/front-end-development-certificate.json

```diff
@@ -240,12 +240,12 @@
       ],
       "type": "Waypoint",
       "challengeType": 7,
-      "nameEs": "Reclama tu certificado de desarrollo Front End",
+      "nameEs": "Reclama tu certificado de Desarrollo de interfaces",
```

Comment:
I don't know but in spanish is called "Desarrollo Front end", I think the name "Desarrollo de interfaces" Could be confused. BTW that should be "Reclama tu certificado de desarrollor Front End"